physics/rigid_motion.hpp

```diff
@@ -76,10 +78,35 @@ RigidMotion<FromFrame, ToFrame> operator*(
     RigidMotion<ThroughFrame, ToFrame> const& left,
     RigidMotion<FromFrame, ThroughFrame> const& right);
 
+// A |RigidTransformation|, its first derivative (a |RigidMotion|), and its
+// second derivative (angular and linear accelerations).
+template<typename FromFrame, typename ToFrame>
+class SecondOrderRigidMotion {
```

Comment:
I'd call this `RigidAcceleration` because that's pretty much what it is and it seems to convey the right meaning.